### PR TITLE
Update API docs

### PR DESCRIPTION
**Pull request by ayeshLK**
## Examples
N/A

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/commons.bal [hunk 1]
```diff
@@ -268,23 +270,49 @@ public type Acknowledgement record {
     *CommonResponse;
 };
 
-# Common responses to be used in the hub implementation.
+# Common response which could be used for `websubhub:TopicRegistrationSuccess`.
```
Comment (praneesha) [inline ballerina/commons.bal:273]:
```suggestion
# Common response, which could be used for `websubhub:TopicRegistrationSuccess`.
```
Please fix all occurrences.

File: ballerina/commons.bal [hunk 2]
```diff
@@ -268,23 +270,49 @@ public type Acknowledgement record {
...
 public final BadUnsubscriptionError BAD_UNSUBSCRIPTION_ERROR = error BadUnsubscriptionError("Bad unsubscription request");
+
+# Common response which could be used for `websubhub:InternalUnsubscriptionError`.
 public final InternalUnsubscriptionError INTERNAL_UNSUBSCRIPTION_ERROR = error InternalUnsubscriptionError("Internal error occurred while processing unsubscription request");
+
+# Common response which could be used for `websubhub:UnsubscriptionDeniedError`.
 public final UnsubscriptionDeniedError UNSUBSCRIPTION_DENIED_ERROR = error UnsubscriptionDeniedError("Unsubscription denied");
 
-# Record to represent client configuration for HubClient / PublisherClient
+# Record to represent client configuration for HubClient / PublisherClient.
```
Comment (praneesha) [inline ballerina/commons.bal:315]:
```suggestion
# Record to represent the client configuration for the HubClient/PublisherClient.
```

File: ballerina/hub_client.bal [hunk 3]
```diff
@@ -38,7 +38,7 @@ public client class HubClient {
     # });
     # ```
     #
-    # + subscription - Original subscription details for the `subscriber`
+    # + subscription - Original `websubhub:Subscription` record which contains the details of the `subscriber`
```
Comment (praneesha) [inline ballerina/hub_client.bal:41]:
```suggestion
    # + subscription - Original `websubhub:Subscription` record, which contains the details of the `subscriber`
```

File: ballerina/commons.bal [hunk 4]
```diff
@@ -268,23 +270,49 @@ public type Acknowledgement record {
     *CommonResponse;
 };
 
-# Common responses to be used in the hub implementation.
+# Common response which could be used for `websubhub:TopicRegistrationSuccess`.
 public final readonly & TopicRegistrationSuccess TOPIC_REGISTRATION_SUCCESS = {};
+
+# Common response which could be used for `websubhub:TopicRegistrationError`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:276]:
```suggestion
# Common response, which could be used for `websubhub:TopicRegistrationError`.
```

File: ballerina/commons.bal [hunk 5]
```diff
@@ -268,23 +270,49 @@ public type Acknowledgement record {
...
 public final TopicRegistrationError TOPIC_REGISTRATION_ERROR = error TopicRegistrationError("Topic registration failed");
+
+# Common response which could be used for `websubhub:TopicDeregistrationSuccess`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:279]:
```suggestion
# Common response, which could be used for `websubhub:TopicDeregistrationSuccess`.
```

File: ballerina/commons.bal [hunk 6]
```diff
@@ -268,23 +270,49 @@ public type Acknowledgement record {
...
 public final readonly & TopicDeregistrationSuccess TOPIC_DEREGISTRATION_SUCCESS = {};
+
+# Common response which could be used for `websubhub:TopicDeregistrationError`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:282]:
```suggestion
# Common response, which could be used for `websubhub:TopicDeregistrationError`.
```

File: ballerina/commons.bal [hunk 7]
```diff
@@ -268,23 +270,49 @@ public type Acknowledgement record {
...
 public final TopicDeregistrationError TOPIC_DEREGISTRATION_ERROR = error TopicDeregistrationError("Topic deregistration failed!");
+
+# Common response which could be used for `websubhub:Acknowledgement`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:285]:
```suggestion
# Common response, which could be used for `websubhub:Acknowledgement`.
```

File: ballerina/commons.bal [hunk 8]
```diff
@@ -268,23 +270,49 @@ public type Acknowledgement record {
...
 public final readonly & Acknowledgement ACKNOWLEDGEMENT = {};
+
+# Common response which could be used for `websubhub:UpdateMessageError`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:288]:
```suggestion
# Common response, which could be used for `websubhub:UpdateMessageError`.
```

File: ballerina/commons.bal [hunk 9]
```diff
@@ -268,23 +270,49 @@ public type Acknowledgement record {
...
 public final UpdateMessageError UPDATE_MESSAGE_ERROR = error UpdateMessageError("Error in accessing content");
+
+# Common response which could be used for `websubhub:SubscriptionAccepted`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:291]:
```suggestion
# Common response, which could be used for `websubhub:SubscriptionAccepted`.
```

File: ballerina/commons.bal [hunk 10]
```diff
@@ -268,23 +270,49 @@ public type Acknowledgement record {
...
 public final readonly & SubscriptionAccepted SUBSCRIPTION_ACCEPTED = {};
+
+# Common response which could be used for `websubhub:BadSubscriptionError`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:294]:
```suggestion
# Common response, which could be used for `websubhub:BadSubscriptionError`.
```

File: ballerina/commons.bal [hunk 11]
```diff
@@ -268,23 +270,49 @@ public type Acknowledgement record {
...
 public final BadSubscriptionError BAD_SUBSCRIPTION_ERROR = error BadSubscriptionError("Bad subscription request");
+
+# Common response which could be used for `websubhub:InternalSubscriptionError`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:297]:
```suggestion
# Common response, which could be used for `websubhub:InternalSubscriptionError`.
```

File: ballerina/commons.bal [hunk 12]
```diff
@@ -268,23 +270,49 @@ public type Acknowledgement record {
...
 public final InternalSubscriptionError INTERNAL_SUBSCRIPTION_ERROR = error InternalSubscriptionError("Internal error occurred while processing subscription request");
+
+# Common response which could be used for `websubhub:SubscriptionDeniedError`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:300]:
```suggestion
# Common response, which could be used for `websubhub:SubscriptionDeniedError`.
```

File: ballerina/commons.bal [hunk 13]
```diff
@@ -268,23 +270,49 @@ public type Acknowledgement record {
...
 public final SubscriptionDeniedError SUBSCRIPTION_DENIED_ERROR = error SubscriptionDeniedError("Subscription denied");
+
+# Common response which could be used for `websubhub:UnsubscriptionAccepted`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:303]:
```suggestion
# Common response, which could be used for `websubhub:UnsubscriptionAccepted`.
```

File: ballerina/commons.bal [hunk 14]
```diff
@@ -268,23 +270,49 @@ public type Acknowledgement record {
...
 public final readonly & UnsubscriptionAccepted UNSUBSCRIPTION_ACCEPTED = {};
+
+# Common response which could be used for `websubhub:BadUnsubscriptionError`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:306]:
```suggestion
# Common response, which could be used for `websubhub:BadUnsubscriptionError`.
```

File: ballerina/commons.bal [hunk 15]
```diff
@@ -268,23 +270,49 @@ public type Acknowledgement record {
...
 public final BadUnsubscriptionError BAD_UNSUBSCRIPTION_ERROR = error BadUnsubscriptionError("Bad unsubscription request");
+
+# Common response which could be used for `websubhub:InternalUnsubscriptionError`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:309]:
```suggestion
# Common response, which could be used for `websubhub:InternalUnsubscriptionError`.
```

File: ballerina/commons.bal [hunk 16]
```diff
@@ -268,23 +270,49 @@ public type Acknowledgement record {
...
 public final InternalUnsubscriptionError INTERNAL_UNSUBSCRIPTION_ERROR = error InternalUnsubscriptionError("Internal error occurred while processing unsubscription request");
+
+# Common response which could be used for `websubhub:UnsubscriptionDeniedError`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:312]:
```suggestion
# Common response, which could be used for `websubhub:UnsubscriptionDeniedError`.
```

**Review by praneesha — APPROVED** [review]

**Pull request merged**